packages/init/README.md

```diff
@@ -25,7 +25,10 @@ const init = require("@webpack-cli/init").default;
 init();
 
 // we're slicing node.process, ...myPacakges is a webpack-scaffold name/path
-init([null, null, ...myPacakges]);
```

Comment:
We actually need to support both, so this change needs to be adjusted accordingly

Reply:
support both ?
cli and node ?

Reply:
No we're supporting both auto flag and multiple pkgs

Reply:
```js
options = [...myPackages,"--auto"]
init(null, null, null, options);
```
like this can we add in the docs ?

Reply:
Let's keep the array convention, and stick with either having a spread plus the auto flag or just the packages.

Reply:
> stick with either having a spread plus the auto flag or just the packages.

thinking of keeping with the auto flag. 
 
> Let's keep the array convention, 

but then in the init method , the method argument will be an array of array. of slicing of 3 may not work

Reply:
Then you can tweak the api